__tests__/screens/FeedScreen/Content.test.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import {render, cleanup} from '@testing-library/react-native';
+import {SafeAreaProvider} from 'react-native-safe-area-context';
 
 import Content from '../../../src/screens/FeedScreen/Content';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. This is not a substantive issue as it doesn't affect the functionality of the code.
```diff
- import {SafeAreaProvider} from 'react-native-safe-area-context';
- import Content from '../../../src/screens/FeedScreen/Content';
+ import {SafeAreaProvider} from 'react-native-safe-area-context';
+ import Content from '../../../src/screens/FeedScreen/Content';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->